src/main.ts

```diff
@@ -82,6 +82,7 @@ async function main() {
 	// - `parentHash`: The hash of the parent block.
 	// - `stateRoot`: The state root after executing this block.
 	// - `extrinsicsRoot`: The Merkle root of the extrinsics.
+	// - `authorId`: The account ID of the block author (may be undefined for development chains).
```

Comment:
If we want this to be generic for Substrate chains, then some production chains might not have have account IDs, although for Polkadot and Kusama this statement is true.
```suggestion
	// - `authorId`: The account ID of the block author (may be undefined for some chains).
```